packages/grid/_modules_/grid/hooks/features/rows/useGridEditRows.ts

```diff
@@ -0,0 +1,240 @@
+import * as React from 'react';
+import {
+  GRID_CELL_MODE_CHANGE,
+  GRID_CELL_CHANGE,
+  GRID_CELL_CHANGE_COMMITTED,
+  GRID_EDIT_ROW_MODEL_CHANGE,
+} from '../../../constants/eventsConstants';
+import { GridApiRef } from '../../../models/api/gridApiRef';
+import { GridEditRowApi } from '../../../models/api/gridEditRowApi';
+import { GridCellMode } from '../../../models/gridCell';
+import { GridEditRowsModel, GridEditRowUpdate } from '../../../models/gridEditRowModel';
+import { GridFeatureModeConstant } from '../../../models/gridFeatureMode';
+import { GridRowId } from '../../../models/gridRows';
+import { GridCellParams } from '../../../models/params/gridCellParams';
+import {
+  GridCellModeChangeParams,
+  GridEditCellParams,
+  GridEditRowModelParams,
+} from '../../../models/params/gridEditCellParams';
+import { buildGridCellParams } from '../../../utils/paramsUtils';
+import { useGridApiEventHandler } from '../../root/useGridApiEventHandler';
+import { useGridApiMethod } from '../../root/useGridApiMethod';
+
+import { optionsSelector } from '../../utils/optionsSelector';
+import { useGridSelector } from '../core/useGridSelector';
+import { useGridState } from '../core/useGridState';
+
+export function useGridEditRows(apiRef: GridApiRef) {
```

Comment:
I have a more general question - what is more appropriate for naming convention useGridEditRows or useEditGridRows, basically if there is a verb in the name should it be before Grid or after it?

Reply:
before `useGridAttackSomething` :)

Reply:
I like the `useGrid` prefix too 👍 , it will be easier to search in the public API once we make them public.